test/integration_test/basic_test.go

```diff
@@ -42,6 +42,18 @@ var testStorageClusterBasicCases = []types.TestCase{
 		}),
 		TestFunc: BasicInstall,
 	},
+	{
+		TestName:        "InstallInCustomNamespaceWithShiftedPort",
+		TestrailCaseIDs: []string{"C52411", "C52430", "C53572"},
+		TestSpec: ci_utils.CreateStorageClusterTestSpecFunc(&corev1.StorageCluster{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "custom-ns-shifted-port",
+				Namespace: "custom-namespace",
+			},
+			Spec: corev1.StorageClusterSpec{StartPort: func(val uint32) *uint32 { return &val }(17001)},
+		}),
+		TestFunc: BasicInstall,
```

Comment:
We need to validate that the [portworx proxy component](https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/component/portworx_proxy.go) is created when custom port and namespaces are used. We can add that validation in the ValidateStorageCluster function itself.